sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -78,7 +78,7 @@ def body(self) -> bytes:
         """
         return self.internal_response.content
 
-    def stream_download(self, pipeline: Pipeline, **kwargs) -> Iterator[bytes]:
+    def stream_download(self, pipeline: Pipeline, *, decompress:bool = True, **kwargs) -> Iterator[bytes]:
```

Comment:
This should be failing the black check right (lack of space after `:`)? I am curious how this is slipping through. Seems like our Analyze step is only checking `azure-core` for some reason.

Reply:
I can run black on the changes

Reply:
@scbedd